### PR TITLE
Refactor/responsiveness

### PR DESCRIPTION
**Pull request by ConnorAndersonLarson**
# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?

- [X] Fix
- [ ] Feature
- [X] Refactor
- [X] Style
- [ ] Testing

### Are there any known bugs?

- [X] No
- [ ] Yes (if so please disclose in Notes for Reviewer & Next Iterations)

### Summary of the changes
Returns text color change on planet hover (main page)
Adds planet scaling on hover (main page)
Updates responsiveness formatting
Adds borders to planet tiles when in mobile/tablet views (for accessibility)

### What ticket(s) do the changes resolve?

Please Link Issues to this PR as necessary.

### Notes for the Reviewer
N/A

### How to Test Changes?
Check layout of site across different screen sizes, hover over planet tiles on desktop view

### CODE REVIEW

File: src/components/Header/Header.css [hunk 1]
```diff
@@ -18,3 +18,18 @@
...
+  .title {
+    margin: 0 50px 0 0 ;
+  }
+}
+
+@media only screen and (max-width: 580px) {
+  .title {
+    margin: 7.5px;
+  }
+}
```
Comment (knees4bees) [inline src/components/Header/Header.css:35]: Great improvements. I like that the title moves to the right when the screen gets smaller.

File: src/components/Planet/planet.css [hunk 2]
```diff
@@ -17,6 +17,11 @@
   cursor: pointer;
 }
 
+.planet-icon:hover {
+  transform: scale(1.05);
+  transition: transform 1s ease-in-out;
+}
+
```
Comment (knees4bees) [inline src/components/Planet/planet.css:24]: Boom

File: src/components/Planet/planet.css [hunk 3]
```diff
@@ -44,6 +50,20 @@
...
+  #neptune,
+  #uranus,
+  #mercure {
+    border: 2px solid #288FDE;
+  }
+  #venus,
+  #mars,
+  #jupiter,
+  #saturne {
+    border: 2px solid #F04528;
```
Comment (knees4bees) [inline src/components/Planet/planet.css:66]: Yes! The colored text is back!

**Review by knees4bees — APPROVED** [review]
Nice improvements!

File: src/components/Header/Header.css (hunk 1 above)
Comment (alexthompson207) [reply src/components/Header/Header.css:35]: Echoing above ^ looks great!

File: src/components/Planet/planet.css (hunk 2 above)
Comment (alexthompson207) [reply src/components/Planet/planet.css:24]: Wahoooo

File: src/components/Planet/planet.css [hunk 4]
```diff
@@ -44,6 +50,20 @@
 @media only screen and (max-width: 850px) {
   .planet-card {
     width: 30%;
+    padding: 10px;
+    border-radius: 35px;
+  }
+  #terre,
+  #neptune,
```
Comment (alexthompson207) [inline src/components/Planet/planet.css:57]: I'm sorry you had to do this twice, but it looks great!

**Review by alexthompson207 — APPROVED** [review]

**Pull request merged**